App.js

```diff
@@ -73,10 +74,11 @@ class Content extends React.Component {
 }
 
 export default function App() {
-
+    const dimensions = useWindowDimensions();
```

Comment:
Is it possible to put those lines of code into the render function of the main `Content` component? 

So the sidebar can dynamically changed when user is changing the window size?

Reply:
React doesn't allow that. I tried it and it complained saying that's not how react works